### PR TITLE
chore: auto request all reviewers

### PR DESCRIPTION
**Pull request by rootulp**
Closes https://github.com/celestiaorg/celestia-app/issues/4092

Note: hasn't been tested yet. Seems like something we can verify post merge.

### CODE REVIEW

File: .github/auto_request_review.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # More info at https://github.com/necojackarc/auto-request-review
 reviewers:
-  # The default reviewers
   defaults:
-    # Example of Github Team. Github team must have write access to repo.
-    # NOTE: This assigned the team itself, not members of the team. The Github
-    # team auto PR assignment will then turn this into individuals
-    - team:celestia-core # This is the Github Team
+    - cmwaters
```
Comment (rootulp) [inline .github/auto_request_review.yml:4]: @cmwaters LMK if you would like to be removed from this list. If that's the case, we can explicitly add you to PRs that need your attention

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -9,10 +9,3 @@
 
 # global owners
 * @celestiaorg/celestia-core
-
-# directory owners
-# NOTE: the directory owners should include the global owners unless the global
-# owner is fully deferring ownership to the directory owner
-docs @liamsi @celestiaorg/celestia-core
-specs @liamsi @celestiaorg/celestia-core
-x/blobstream @rach-id @evan-forbes
```
Comment (rootulp) [inline .github/CODEOWNERS:18]: I removed these b/c I think they outlived their purpose

**Review by rach-id — APPROVED** [review]
Approving but Im wondering if there was a decision to make all reviewers review all PRs?

**Comment by rootulp** [comment]
> Approving but Im wondering if there was a decision to make all reviewers review all PRs?

There wasn't a decision so this is up for debate. Currently we have a rule that each PR must have 2 approvals before being merged. Since 2 random ppl get tagged on each PR, it can take a while for those two people to review. I usually add all core/app engineers as reviewers on my PRs so I figured I'd automate it.

File: .github/auto_request_review.yml (hunk 1 above)
Comment (cmwaters) [reply .github/auto_request_review.yml:4]: I'm okay with this for now. I'm not too sensitive to github notifications

**Review by cmwaters — APPROVED** [review]
I prefer everyone being tagged for reviews. Everyone that has the capacity should spend time reviewing everyone else's work. It's super important.

**Pull request merged**